edm.yaml

```diff
@@ -47,250 +52,181 @@ datatypes :
     Description : "Event Info Data"
     Author : "C. Bernet, B. Hegner"
     Members :
-     - int Number // Event number
+     - int number // Event number
 
   fcc::Particle :
     Description : "Reconstructed particle"
     Author : "C. Bernet, B. Hegner"
     Members :
-     - fcc::BareParticle Core // Contains basic particle information.
-
-  fcc::ParticleTrackAssociation:
-    Description: "Association between a Particle and a Track."
-    Author : "C. Bernet, B. Hegner"
-    OneToOneRelations:
-     - fcc::Particle Particle // The particle
-     - fcc::Track Track // The track
-
-  fcc::ParticleClusterAssociation:
-    Description: "Association between a Particle and a Cluster."
-    Author : "C. Bernet, B. Hegner"
-    OneToOneRelations:
-     - fcc::Particle Particle // The particle
-     - fcc::CaloCluster Cluster // The cluster
-
-  fcc::ParticleTagAssociation :
-    Description : "Association between a Particle and Tag information using a preferred algorithm"
-    Author : "C. Helsens"
+     - fcc::BareParticle core // Contains basic particle information.
+    OneToManyRelations:
+     - fcc::Track tracks // The Tracks used in reconstruction
+     - fcc::CaloCluster clusters // The cluster used for reconstruction
+
+  fcc::TaggedParticle :
+    Description : "Association between a Particle and some arbitrary Tag information"
+    Author : "C. Helsens, J. Lingemann"
+    Members:
+     - float value // the value of the tag
```

Comment:
Do we need the `int` variety of tags? For what I understand `float` can cover both use cases.